blog/_posts/2021-08-09-systematics-normalization-blog.md

```diff
@@ -0,0 +1,58 @@
+title: "Systematics Normalization"
+date: 2021-08-10
+author: Abigail Wilson
+---
+
+# Systematics Normalization
+I collaborated on this project with my mentor, Emily Dolson (www.emilyldolson.com). This project is part of [Empirical](https://github.com/devosoft/Empirical). This post will serve as a continuation of my post from last summer with some updated information. For more in depth information into earlier parts of this project and the details about the null model and diversity model, visit my blogpost from last summer here: http://mmore500.com/waves/blog/systematics-blogpost.html
+
+### **Goal and Motivation:**
+
+Phylogenetic tree analysis is a crucial tool for understanding evolutionary history. When measuring phylogenies of different sizes and shapes, it is important to have viable metrics that can be used for comparison. However, many phylogeny measurements are very sensitive to tree size. For example, phylogenetic diversity gets inflated as phylogenies get larger, making it difficult to meaningfully compare over time or between different populations. This problem is exacerbated in the context of digital evolution, where many assumptions about reconstructed phylogenies from nature are violated. It's unclear whether traditional approaches to normalizing phylogeny metrics remain valid in this new context. The optimal end result of this project, and ultimate motivation, is to provide computational and evolutionary biologists with tools that provide meaningful tree metrics. 
+
+The aim of my work this summer has been to provide a continuation of the systematics normalization project I began last summer. The goal of this project has been to provide meaningful metrics for phylogenies of different sizes. Much of my work this summer to meet this goal involved testing previous findings against existing models, fleshing out previous experimentation, and formally recording these findings in a research paper.
+
+### **The Short and Sweet**
+
+The quick overview of my project can be summed up in the following. There is a random tree model which just creates randomly generated trees with whatever specifications for size and age you want. It creates these trees thousands of times and each time a tree is created, metrics about it are recorded and stored. After thousands of runs for trees of different sizes and ages, we look at that data and ask, what are the min and max values for this data and where does everything in between lie? We call this a percentile range. After that, we want to test this percentile range, so we run the model again with different random seeds and some other settings and make sure that they always average in the 50th percentile. Then, we make new models and test where those models fall in our percentile range and how those results fit into our other findings. Eventually, this allows us to confidently return to you both a value for your tree metric but also a percentile for that value. Aka, a useful metric for tree comparison. 
```

Comment:
Perhaps the line, 'test this percentile range' is a little bit vague here. What does it mean for a percentile range to be good? When do we know we found a good one?